docker/build.bash

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+# Copyright 2024 Adevinta
+
+set -e -u
+
+cd "$(dirname $0)/.."
+
+MDBOOK_VERSION=${MDBOOK_VERSION:-v0.4.40}
```

Comment:
I think this variable could be set in the same way as LAVA_VERSION

Reply:
Fixed.